src/interfaces/node.cpp

```diff
@@ -29,6 +29,7 @@
 #include <ui_interface.h>
 #include <util/system.h>
 #include <validation.h>
+#include <wallet/walletutil.h>
```

Comment:
This makes these redeclarations:
https://github.com/bitcoin/bitcoin/blob/964fe494a68c06490fba2e27ad924a4f67b6ae5f/src/interfaces/node.cpp#L43-L44
redundant.

Reply:
Removed include.